src/main/java/core/basesyntax/service/InvalidDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class InvalidDataException extends RuntimeException {
+    public InvalidDataException() {
+        super("Invalid data");
+    }
```

Comment:
Consider adding additional constructors to your `InvalidDataException` class to allow for more detailed messages. For example, you might want to pass a specific message about what exactly is invalid in the data when throwing this exception. This would greatly improve the debugging experience and clarity of your error handling.